### PR TITLE
Kd0aij alt offset fixed

### PR DESCRIPTION
**Pull request by AndreasAntener**
Supersedes #33 
- extended change for export format
- loading alt offset

### CODE REVIEW

File: src/me/drton/flightplot/export/AbstractTrackExporter.java [hunk 1]
```diff
@@ -36,7 +36,11 @@ private void initAnalyzers() {
 
     protected TrackPoint readNextPoint() throws IOException, FormatErrorException {
         TrackPoint point = this.trackReader.readNextPoint();
+        float altOffset = configuration.getAltOffset();
         if (null != point) {
+            if (altOffset != 0) {
```
Comment (DrTon) [inline src/me/drton/flightplot/export/AbstractTrackExporter.java]: Not very useful micro-optimization IMO. Remove please.

File: src/me/drton/flightplot/export/KmlExportFormat.java [hunk 2]
```diff
@@ -11,13 +11,11 @@ public String getFormatName() {
...
     public String getFileExtension() {
-        return ".kml";
+        return "kml";
     }
 
     @Override
-    public String getFileExtensionName() {
-        return "KML";
-    }
+    public String getFileExtensionName() { return "KML"; }
```
Comment (DrTon) [inline src/me/drton/flightplot/export/KmlExportFormat.java]: It's better to be consistent. Don't use one-line functions please.

**Comment by AndreasAntener** [comment]
Updated

**Comment by kd0aij** [comment]
@AndreasAntener Thanks for cleaning up my commits and getting them into master.
Questions about the IDEA project file: 
does mine differ from master because I'm building an executable jar file? 
And if so, what is the correct way to build the executable jar?

**Comment by AndreasAntener** [comment]
@kd0aij That could be the case. FlightPlot has an Ant build.xml which is currently the correct way to build the jar.
IntelliJ has an Ant Build view where you can execute the targets (`flightplot` will build the jar). If you have Ant installed it can also be done from the CLI.

**Comment by DrTon** [comment]
Doing massive rework of export functionality. PR will be ready soon.

**Comment by DrTon** [comment]
Included in #45, thanks!

**Pull request closed without merging**